src/cache.ts

```diff
@@ -26,12 +26,12 @@ export async function setCachedFile(fileName: string, value: Buffer): Promise<vo
 }
 
 export async function getCachedFile(fileName: string, encoding?: BufferEncoding): Promise<Buffer> {
-	let cachedFile: Buffer = Buffer.alloc(0);
+	let cachedFile = Buffer.alloc(0);
```

Comment:
```suggestion
	let cachedFile: Buffer;
```
since cachedFile is always reassigned.

Reply:
`cachedFile` is not always reassigned. If Redis is being used and there is no value for the key, the `cachedFile` is not updated. Using `Buffer.alloc(0)` ensured it always had a value, of some port, in these cases

It's possible this whole pattern should be changed, but that might be better off in another PR